src/components/TextEditor.vue

```diff
@@ -380,11 +410,42 @@ export default {
 				throw new Error('Impossible to execute a command before editor is ready.')
 			}
 		},
+		insertTextBlock(textBlock, addTriggrer = true) {
+			if (addTriggrer) {
+				this.appendToBodyAtCursor('!')
+			}
+			let content = textBlock.content
+			if (!this.html) {
+				const text = new Text('html', content)
+				content = toPlain(text).value
+			}
+			this.editorInstance.execute('insertItem', { content, isHtml: this.html }, '!')
+		},
 	},
 }
 </script>
 
 <style lang="scss" scoped>
+
+.bordered {
```

Comment:
nit: use BEM and a modifier, like `text-editor--bordered`

Reply:
Apologies that I need to reopen this. But for my understanding, is *bordered* not a modifier? https://getbem.com/naming/

I'm absolutely nitpicking